package.json

```diff
@@ -79,25 +80,28 @@
     "publish": "vsce publish patch"
   },
   "devDependencies": {
+    "@elm-tooling/tree-sitter-elm": "github:elm-tooling/tree-sitter-elm#0aecfbf69f0c9bcdfada0f6f2c5398235c23949f",
     "@types/mocha": "^2.2.42",
     "@types/node": "^8.10.25",
+    "@types/vscode": "~1.58.0",
     "@typescript-eslint/eslint-plugin": "^5.20.0",
     "@typescript-eslint/parser": "^5.20.0",
+    "@vscode/test-electron": "^2.1.3",
+    "electron-rebuild": "^2.3.5",
     "eslint": "^8.13.0",
     "eslint-config-prettier": "^8.5.0",
-    "electron-rebuild": "^2.3.5",
     "tree-sitter-agda": "github:pokey/tree-sitter-agda#e5fba6cabe8c7fc7993ced2b86704f3841215284",
     "tree-sitter-bash": "^0.19.0",
     "tree-sitter-c": "^0.19.0",
-    "tree-sitter-css": "^0.19.0",
     "tree-sitter-c-sharp": "^0.19.0",
     "tree-sitter-cli": "^0.20.1",
     "tree-sitter-clojure": "github:sogaiu/tree-sitter-clojure#master",
     "tree-sitter-cpp": "^0.19.0",
-    "@elm-tooling/tree-sitter-elm": "github:elm-tooling/tree-sitter-elm#0aecfbf69f0c9bcdfada0f6f2c5398235c23949f",
-    "tree-sitter-go": "github:tree-sitter/tree-sitter-go#bbaa67a180cfe0c943e50c55130918be8efb20bd",
+    "tree-sitter-css": "^0.19.0",
     "tree-sitter-elixir": "github:elixir-lang/tree-sitter-elixir#b20eaa75565243c50be5e35e253d8beb58f45d56",
+    "tree-sitter-go": "github:tree-sitter/tree-sitter-go#bbaa67a180cfe0c943e50c55130918be8efb20bd",
     "tree-sitter-haskell": "github:tree-sitter/tree-sitter-haskell#d6ccd2d9c40bdec29fee0027ef04fe5ff1ae4ceb",
+    "tree-sitter-hcl": "https://github.com/MichaHoffmann/tree-sitter-hcl",
```

Comment:
please pint to a specific sha. tree-sitter grammars regularly make breaking changes 😅